<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookRefereDialog</class>
 <widget class="QDialog" name="BookRefereDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>تعديل العزو</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolAdd">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../share/moltaqa-lib/qt_rc.qrc">
         <normaloff>:/images/add2.png</normaloff>:/images/add2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolEdit">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../share/moltaqa-lib/qt_rc.qrc">
         <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolDelete">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../share/moltaqa-lib/qt_rc.qrc">
         <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../share/moltaqa-lib/qt_rc.qrc"/>
 </resources>
 <connections/>
</ui>
